### ISSUE TITLE
0.7.2 has syntax errors on ruby 1.9.1

### ISSUE THREAD

**Issue by cweiske** [lfittl/pg_query#44]
I'm running Ubuntu 14.04, the current LTS version.

Running `sqlint` that uses pg_query fails:

    /var/lib/gems/1.9.1/gems/pg_query-0.7.2/lib/pg_query/deparse.rb:2:in
     `require_relative': /var/lib/gems/1.9.1/gems/pg_query-0.7.2/lib/pg_query/deparse/alter_table.rb:31:
     syntax error, unexpected tLPAREN_ARG, expecting keyword_do_LAMBDA or tLAMBEG (SyntaxError)
    ... many more of this

See http://ruby-journal.com/becareful-with-space-in-lambda-hash-rocket-syntax-between-ruby-1-dot-9-and-2-dot-0/ for more info about this.

It'd be nice if you could make this lib compatible with ruby 1.9.

**Comment by lfittl** [comment]
@cweiske Hmm, good point - let me see if I can make Ruby 1.9 work over the holidays.

If its purely based on the hash syntax it should be possible to revert this to 1.9 style without problems. I'll update this issue once I know more.

**Comment by dkolbly** [comment]
FWIW, I was able to get it to load by removing the space between the `->` and `(`, like so:
```
        AT_AddColumn                 => ->(_node) { ['ADD COLUMN'] },
        AT_ColumnDefault             => ->(node) { ['ALTER COLUMN', node['def'] ? 'SET DEFAULT' : 'DROP DEFAULT'] },
```
etc.

**Comment by lfittl** [comment]
@dkolbly @cweiske This has been fixed in 0.9.1 and will from now on be tested on Travis as well. Sorry about the inconvenience!
